src/twisted/internet/defer.py

```diff
@@ -53,6 +53,13 @@ class TimeoutError(Exception):
 
 
 
+class NoCurrentResult(BaseException):
```

Comment:
By documentation do you just mean a comment, or do you explicitly want documentation that would show up on the website?